apps/climatemappedafrica/src/components/Hero/Hero.js

```diff
@@ -84,9 +85,9 @@ function Hero({
               {...props}
             />
             <RichTypography
-              variant="subtitle1"
+              variant="caption"
               sx={{
-                fontSize: "11px",
+                fontSize: { xs: "11px" },
```

Comment:
Doesn't this mean `fontSize` will **always** be `11px`?

Reply:
...because of how it's defined in [`theme`](https://github.com/CodeForAfrica/ui/blob/5fc23720dad9d24379227bc2a5edbe2638200bd5/apps/climatemappedafrica/src/theme/index.js#L233) @koechkevin